frontend/src/components/network-import-form/network-import-form.tsx

```diff
@@ -26,11 +22,7 @@ interface NetworkImportFormProps {
 }
 
 export function NetworkImportForm({ onComplete }: NetworkImportFormProps) {
-  const [showOverwriteCheckbox, setShowOverwriteCheckbox] =
-    React.useState(false)
-  const {
-    state: { networks, presets }
-  } = useGlobal()
+  const [showOverwriteCheckbox, setShowOverwriteCheckbox] = useState(false)
```

Comment:
suggestion(non-blocking): nothing major, but conventionally you would use "is" for  binary state so e.g isCheckboxVisible

Reply:
changed